tests/php/modules/publicize/test_class.publicize.php

```diff
@@ -74,12 +76,18 @@ public function test_publicize_does_not_fire_on_post_types_that_do_not_support_i
 	function assertPublicized( $should_have_publicized, $post ) {
 		if ( $should_have_publicized ) {
 			$this->assertTrue( $this->fired_publicized_post );
-			$this->assertEquals( $post->ID, $this->publicized_post_id );	
+			$this->assertEquals( $post->ID, $this->publicized_post_id );
+			$this->assertTrue( $this->in_publish_filter );
 		} else {
 			$this->assertFalse( $this->fired_publicized_post );
 			$this->assertNull( $this->publicized_post_id );
+			$this->assertFalse( $this->in_publish_filter );
 		}
-		
+	}
+
+	function _is_in_flags( $flags, $post ) {
```

Comment:
I think it adds confusion the fact that an `is_` method sets an object attribute.